### PR TITLE
Update Japanese translation to 75fafe1

### PR DESCRIPTION
**Pull request by 8398a7**

### CODE REVIEW

**Comment by 8398a7** [comment]
:+1:

File: docs/docs/10.9-perf.ja-JP.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Reactは普通、従来の枠を超えてとても速いです。しかし、ア
 ### `Perf.start()` と `Perf.stop()`
 測定の開始/終了です。その間のReactの操作は以下のような分析のために記録されます。あまり時間を使わない操作は無視されます。
 
+停止した後、あなたは、測定結果を得るために`Perf.getLastMeasurements（）`（後述）が必要になります。
```
Comment (zpao) [inline docs/docs/10.9-perf.ja-JP.md]: Can you fix the code piece a bit here to remove the extraneous whitespace (around the parens) and put a space before the opening backtick?

Comment (8398a7) [inline docs/docs/10.9-perf.ja-JP.md]: Sorry, I fixed it.

**Comment by zpao** [comment]
Merged in c6ecda7503ecac374f941fdf7af75755b467f449 (made a small change to fix the paren characters), thanks!

**Pull request closed without merging**